dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/tests/VisibilityTest.java

```diff
@@ -732,10 +732,13 @@ public void testTolerates() {
             // MicroProfile features do not currently follow this convention.  They may need to in the future.
             // openapi features are stabilized and were not updated to support EE 9.
             // opentracing is also now stabilized and does not support running with EE 10+
-            if (featureInfo.getVisibility().equals("public") && (featureName.startsWith("io.openliberty.mp") || featureName.startsWith("com.ibm.websphere.appserver.mp")
+            // MicroProfile 7.0 tolerates both EE10 and EE11
+            if (featureInfo.getVisibility().equals("public") && (featureName.startsWith("io.openliberty.mp")
+                                                                 || featureName.startsWith("com.ibm.websphere.appserver.mp")
                                                                  || featureName.startsWith("com.ibm.websphere.appserver.opentracing")
                                                                  || featureName.startsWith("com.ibm.websphere.appserver.openapi")
-                                                                 || featureName.startsWith("com.ibm.websphere.appserver.microProfile-1."))) {
+                                                                 || featureName.startsWith("com.ibm.websphere.appserver.microProfile-1.")
+                                                                 || featureName.startsWith("io.openliberty.microProfile-7."))) {
                 continue;
             }
```

Comment:
Noting the comment at the top of this method:
```java
    /**
     * This test validates that jakarta and value-add features do not have public or protected tolerated features so that versionless
     * features and product extension / user features that have tolerates in their features for all ee levels can do so and avoid
     * having to do an auto feature.
     */
```

Do we know what the design is for MP 7.0 features and versionless? This test seems to state a requirement on our jakarta and value add features (which includes almost everything). Are we happy that that requirement shouldn't apply to microprofile features?